css/reveal-override.css

```diff
@@ -1,75 +1,98 @@
-@import url(//fonts.googleapis.com/css?family=Cabin:400,700);
-@import url(//fonts.googleapis.com/css?family=Special+Elite:400,700);
+@import url(http://fonts.googleapis.com/css?family=Catamaran:400,700);
```

Comment:
by accident ... did it only for my presentation but it got mixed into the first pull request. Already fixed the commit.